usr.sbin/pkg/Makefile

```diff
@@ -9,10 +9,15 @@ CONFS=		${PKGCONF}
 CONFSNAME_${PKGCONF}=	${PKGCONF:C/\.conf.+$/.conf/}
 CONFSDIR=	/etc/pkg${PKG_SUFFIX}
 CONFSMODE=	644
-PROG=	pkg${PKG_SUFFIX}
+PROG?=	pkg${PKG_SUFFIX}
 SRCS=	pkg.c dns_utils.c config.c
 MAN=	pkg${PKG_SUFFIX}.7
 
+.if ${MACHINE_ABI:Mpurecap} && empty(PKG_SUFFIX)
```

Comment:
```suggestion
.if ${PROG} == "pkg64c"
```

Reply:
Or move this to pkg64c's Makefile if people disagree with me and want that to exist? Then you don't need a condition.

Reply:
Shell script can move there too so it goes away when pkg64c gets named back; other than the PROG?= change that'd revert everything back to what you want for pkg+pkg64 rather than pkg64c+pkg64

Reply:
I applied the suggestions and moved pkg.sh to usr.sbin/pkg64c. I like the fact we have only one line changed in usr.sbin/pkg now.